.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "search.usePCRE2": true
+  "search.usePCRE2": true,
+  "git.ignoreLimitWarning": true
```

Comment:
What does this do?

Reply:
Oh, sorry, I didn't see this!

I have too many git repositories (my home directory is a private repository, and I `git submodule add` everything to it --- it makes moving to a new machine an amazingly simple process --- one `git clone` and I'm set. However VSCode tries to tell me about all of them, even when I have a subrepository open (a bug I've never followed up on). There are too many for it to cope with, so occasionally it gives me an error message. This time it had an option "don't tell me about this again", but it seems to have put that in the project `settings.json` file rather than my user file.

Reply:
```suggestion
```